evap/contributor/views.py

```diff
@@ -108,6 +109,7 @@ def index(request):
         "semester_list": semester_list,
         "show_delegated": show_delegated,
         "delegate_selection_form": DelegateSelectionForm(),
+        "infotext": Infotext.objects.get(page=Infotext.Page.CONTRIBUTOR_INDEX),
```

Comment:
I wonder if the infotexts would be better encapsulated if they would be included using [inclusion tags](https://docs.djangoproject.com/en/4.1/howto/custom-template-tags/#inclusion-tags) - there wouldn't be any global `infotext` in the template, instead it would just have `{% include_infobox Infotext.Page.CONTRIBUTOR_INDEX %}` where the infobox should be. The downside would be that it is not clear from just looking at the view function where database accesses happen, however we need to hit the DB anyways, so its not like we would overlook an unnecessary hit. @He3lixxx what do you think?

Reply:
Yeah sounds good.